docs/src/literate-tutorials/reactive_surface.jl

```diff
@@ -0,0 +1,259 @@
+# Putting this flag to false reproduces the figure shown in the example #src
+# We check for laminar flow development in the CI                       #src
+if isdefined(Main, :is_ci) #hide
+    IS_CI = Main.is_ci     #hide
+else                       #hide
+    IS_CI = false          #hide
+end                        #hide
+# # [Reactive Surface](@id tutorial-reactive-surface)
+#
+# ![](reactive_surface.gif)
+#
+# *Figure 1*: Reactant concentration field of the Gray-Scott model on the unit sphere.
+#
+#-
+#md # !!! tip
+#md #     This example is also available as a Jupyter notebook:
+#md #     [`reactive_surface.ipynb`](@__NBVIEWER_ROOT_URL__/examples/reactive_surface.ipynb).
+#-
+#
+# ## Introduction
+#
+# This tutorial gives a quick tutorial on how to assemble and solve time-dependent problems
+# on embedded surfaces.
+#
+# For this showcase we use the well known Gray-Scott model, which is a well-known reaction-diffusion
+# system to study pattern formation. The strong form is given by
+#
+# ```math
+#  \begin{aligned}
+#    \partial_t r_1 &= \nabla \cdot (D_1 \nabla r_1) - r_1*r_2^2 + F *(1 - r_1) \quad \textbf{x} \in \Omega, \\
+#    \partial_t r_2 &= \nabla \cdot (D_2 \nabla r_2) + r_1*r_2^2 - r_2*(F + k ) \quad \textbf{x} \in \Omega,
+#  \end{aligned}
+# ```
+#
+# where $r_1$ and $r_2$ are the reaction fields, $D_1$ and $D_2$ the diffusion tensors,
+# $k$ is the conversion rate, $F$ is the feed rate and $\Omega$ the domain. Depending on the choice of
+# parameters a different pattern can be observed. Please also note that the domain does not have a
+# boundary. The corresponding weak form can be derived as usual.
+#
+# For simplicity we will solve the problem with the Lie-Troter-Godunov operator splitting technique with
+# the classical reaction-diffusion split. In this method we split our problem in two problems, i.e. a heat
+# problem and a pointwise reaction problem, and solve them alternatingly to advance in time.
```

Comment:
Since this is one of the "highlights" of this tutorial as advertised by the snipped in `tutorial/index.md` I think it would be nice to expand this a bit with some equations (e.g. the weak form of the heat equation and the strong form of the reaction problem.

Reply:
Is this sufficient or do you have more in mind?